editor/components/default-block-appender/style.scss

```diff
@@ -1,7 +1,10 @@
 $empty-paragraph-height: $text-editor-font-size * 4;
 
 .editor-default-block-appender {
-	.editor-default-block-appender__content {
+	// Componsate for outline being applied outside the appender
+	// Every other block puts the outline on a :before pseudo element, which places it inside
+
+	input[type=text].editor-default-block-appender__content {
```

Comment:
`input[type=text` qualifier necessary?

Reply:
So, it was when `.editor-default-block-appender__content stood` on its own. But if I prefix `.editor-default-block-appender`, we increase the specificity, so we can remove the qualifier. 

<img width="327" alt="screen shot 2018-03-12 at 10 38 23" src="https://user-images.githubusercontent.com/1204802/37276295-9d49c242-25e1-11e8-98b4-1f73a2e1e116.png">


Incidentally that allowed me to tune up the appender SCSS file a fair bit.